swift-sdk.xcodeproj/project.pbxproj

```diff
@@ -585,6 +594,17 @@
 			path = "swift-sdk-objc-tests";
 			sourceTree = "<group>";
 		};
+		AC819182227138D40014955E /* Diff */ = {
```

Comment:
I think we should move these library files from Dwifft to a separate folder in the filesystem as well.